src/ui/theme/grid.ts

```diff
@@ -1,6 +1,11 @@
 import { PixelRatio } from 'react-native'
 
-export const BORDER_RADIUS = 8
+import { getSpacing } from 'ui/theme/spacing'
+
+export enum BorderRadiusEnum {
```

Comment:
C'est pas une mauvaise pratique de suffixer les enums par `Enum` ?
```suggestion
export enum BorderRadius {
```
En voyant les membres de ton enums, je ne suis pas sûr que `CHECKBOX_RADIUS` soit utilisé.
Tu pourrais juste créer une constante BORDER_RADIUS et l'exporter.

Reply:
Un oublie de ma part le checkbox_radius est bien utilisé maintenant !
Pour le nommage, je me suis basé sur ColorsEnum qui existe comme ça